src/app/home/home.component.ts

```diff
@@ -32,21 +36,36 @@ export class HomeComponent implements OnInit {
      console.log("ok");
      this.submitted = true;
       if (this.loginForm.valid) { 
-        console.log(this.loginForm.value);
-         sessionStorage.setItem('access_token', '234');
-              sessionStorage.setItem('user_id', '123');
-
-              localStorage.setItem('access_token', '234');
-              localStorage.setItem('user_id', '123');
-              let isLoggedIn = (localStorage.getItem('access_token'))?true:false; 
-              //this.isLoggedIn = true;
-              let fetch={
-              user_id:'123',
-              access_token:'234'
-
-              };  
-              location.href = '/tax-prepare-profile'; 
-             // location.href = '/dashboard'; 
+        console.log(this.loginForm.value);         
+        
+        this.authService.login(this.f.email.value, this.f.password.value, this.f.remember_me.value,)
+            .pipe(first())
+            .subscribe(
+                resp => {
+                console.log("###");
+                console.log(resp);
```

Comment:
@Bithika-Redapple Stay away for any short hand. For example, do not say "resp", say "response". If you can add descriptor that is even better. For example, `captchaResponse`. I am noticing a lot of shorthand which is not clear and can be misinterpreted.